packages/terra-application-name/src/ApplicationHeaderName.jsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+
+import AppDelegate from 'terra-app-delegate';
+
+import styles from './ApplicationHeaderName.scss';
+
+const cx = classNames.bind(styles);
+
+const SizeTypes = {
+  TINY: 'tiny',
+  SMALL: 'small',
+  MEDIUM: 'medium',
+  LARGE: 'large',
+  HUGE: 'huge',
+};
+
+const propTypes = {
+  /**
+   * The accessory element to be displayed next to the title.
+   * */
+  accessory: PropTypes.element,
+  /**
+   * The AppDelegate instance provided by the containing component. If present, its properties will propagate to the children components.
+   * */
+  app: AppDelegate.propType,
+  /**
+   * The breakpoint size that the window is currently at. Takes one of: `tiny`, `small`, `medium`, `large`, `huge`.
```

Comment:
Should we list the expected strings or the Utils Keys here?

Reply:
The util keys feel a little cumbersome for some situations, but they do help abstract the internal implementation. 

I think it's easier to read and write (both documentation and code syntax):
```
<ApplicationHeaderName size="tiny" />
```
vs
```
<ApplicationHeaderName size={ApplicationHeaderName.opts.SizeTypes.TINY} />
```
There's advantages and disadvantages to both.

Reply:
+1 to sizes

Reply:
Per offline discussion with @dkasper-was-taken, we'll let the higher order header handle the breakpoint screen sizes. The size prop has been removed, along with the util keys.